scss/_patterns_newsletter-signup.scss

```diff
@@ -0,0 +1,45 @@
+/*
+  @classreference
+  newsletter-signup:
+    Newsletter signup:
+      .p-newsletter-signup:
+        Main element of the newsletter signup component.
+      .p-newsletter-signup--25-75:
+        Modifier class for a 25-75 split layout.
+      .p-newsletter-signup--50-50:
+        Modifier class for a 50-50 split layout.
+      .p-newsletter-signup--2-col:
+        Modifier class for a 2 column layout.
+        Takes at most 2 columns on large and medium dimensions.
+        Takes full width  on small dimensions.
+      .p-newsletter-signup--4-col:
+        Modifier class for a 4 column layout.
+        Takes at most 4 columns on large and medium dimensions.
+        Takes full width  on small dimensions.
+*/
+
+@import 'settings';
+
+@mixin vf-p-newsletter-signup {
+  .p-newsletter-signup--2-col,
+  .p-newsletter-signup--4-col {
+    // same as %section-padding--default
+    padding-bottom: $spv--strip-regular * 0.5;
+
+    @media (min-width: $breakpoint-large) {
+      padding-bottom: $spv--strip-regular;
+    }
+  }
+
+  @media (min-width: $threshold-4-small-4-med-col) {
+    .p-newsletter-signup--2-col {
+      padding-bottom: 0 !important;
+    }
+  }
+
+  @media (min-width: $threshold-4-8-col) {
+    .p-newsletter-signup--4-col {
+      padding-bottom: 0 !important;
```

Comment:
We try to avoid `!important` when we can. Maybe we could use `@media (width <...` queries here to apply the padding more selectively in the first place, so you don't need to use `!important`? This would also help reduce the stylesheet size increase. 

Not a blocking comment